scripts/update/requestrr.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
If the update and the install share large portions of code, I would encourage you to abstract them into a function file you can then source in both the installer and the updater. Less code duplication, one place to keep track of it.

Reply:
did this :)